Makefile

```diff
@@ -83,66 +51,49 @@ test-requirements:
 	         echo "Run 'make freeze-requirements' to update."; exit 1; } \
 || { echo "requirements.txt is up to date"; exit 0; }
 
-define run_docker_container
-	docker run -it --rm \
-		--name "${DOCKER_CONTAINER_PREFIX}-${1}" \
-		-e NOTIFICATION_QUEUE_PREFIX=${NOTIFICATION_QUEUE_PREFIX} \
-		-e NOTIFY_ENVIRONMENT=${NOTIFY_ENVIRONMENT} \
-		-e FLASK_APP=${FLASK_APP} \
-		-e FLASK_DEBUG=${FLASK_DEBUG} \
-		-e WERKZEUG_DEBUG_PIN=${WERKZEUG_DEBUG_PIN} \
-		-e AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
-		-e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \
-		-e NOTIFY_LOG_PATH=${NOTIFY_LOG_PATH} \
-		${3} \
-		${DOCKER_IMAGE_NAME} \
-		${2}
-endef
-
-
 # ---- DOCKER COMMANDS ---- #
 
-.PHONY: run-with-docker
-run-with-docker: prepare-docker-build-image ## Build inside a Docker container
-	$(call run_docker_container,celery-build, make _run)
+.PHONY: bootstrap
+bootstrap: generate-version-file ## Setup environment to run app commands
+	docker build -f docker/Dockerfile --target test -t notifications-antivirus .
 
-.PHONY: run-app-with-docker
-run-app-with-docker: prepare-docker-build-image ## Build inside a Docker container
-	$(call run_docker_container,app-build, make _run_app, -p 6016:6016)
+.PHONY: run-celery
+run-celery: ## Run celery in Docker container
+	$(if ${NOTIFICATION_QUEUE_PREFIX},,$(error Must specify NOTIFICATION_QUEUE_PREFIX))
```

Comment:
Optional - Could consider not checking for `NOTIFICATION_QUEUE_PREFIX` since the app can run locally without it and we don't do `source environment.sh` to get it until line 63

Reply:
I'm open to getting removing it - we have it in [Template Preview](https://github.com/alphagov/notifications-template-preview/blob/master/Makefile#L51), so I thought I'd be consistent. (But people should just read the README!)

I'm a bit confused, though: since Celery needs a broker to connect to, don't we need to specify the queue prefix before we run it locally? (We don't use a local SQS instance, if that's even possible).

Reply:
I didn't realise we had that in template preview - happy to go with what you think's best.

You can start the Celery app up without the queue prefix, but you would still need if it you were trying to call the task from API, so my earlier comment that the app can run locally without it wasn't quite right.